openassessment/xblock/studio_mixin.py

```diff
@@ -151,8 +157,10 @@ def editor_context(self):
             'feedbackprompt': self.rubric_feedback_prompt,
             'feedback_default_text': feedback_default_text,
             'text_response': self.text_response if self.text_response else '',
+            'text_response_editor': self.text_response_editor if self.text_response_editor else '',
```

Comment:
Would it make sense to use the default `text` as [here](https://github.com/edx/edx-ora2/pull/1576/files#diff-76ce3a8b3d4643ee71466f0dc9d6628478b6e76d8c8af079dab15f12cb96b593R882)?

Reply:
Yes. Updated. Thanks for noticing.